downstream/modules/platform/proc-configuring-controller-ldap-security.adoc

```diff
@@ -4,7 +4,7 @@
 Use this procedure to configure LDAP security for your {ControllerName}.
 
 .Procedure
-. Under *LDAP Certificate Authority Trust Bundle* click the drop-down menu and select a secret.
+. Under *LDAP Certificate Authority Trust Bundle* click the drop-down menu and select *ldap-ca.crt*.
```

Comment:
The issue reporter is referring to the format of the information inside the secret itself.  This docs section in the upstream docs describes it.

* https://ansible.readthedocs.io/projects/awx-operator/en/latest/user-guide/advanced-configuration/trusting-a-custom-certificate-authority.html

Something like this note is what they are expecting I think:

> Please note the awx-operator will look for the data field ldap-ca.crt in the specified secret when using the ldap_cacert_secret.

It probably would be good to add an example of how to create the secret with the `oc` create secret` command too.

Reply:
I've updated the doc based on the feedback you provided.

Reply:
Change implemented. Thank you!